entries/jQuery.get.xml

```diff
@@ -22,7 +22,7 @@
     </argument>
   </signature>
   <signature>
-    <added>1.12/2.2</added>
+    <added>1.12-2.2</added>
```

Comment:
Is the dash necessary? This implies that all the versions between 1.12 and 2.2 are involved.

Reply:
I understand the `/` doesn't work, but what about a `+` or space?

Reply:
Neither of these work, unfortunately. :( Neither does `_` or `,`. The checks there are pretty restrictive...

I went for `1.12-and-2.2`. Not pretty but perhaps clearer that what we have now.